### DISCUSSION TITLE
[Need a review] HestiCP & Craft CMS

### DISCUSSION THREAD

**Discussion by anvme** [craftcms/cms#9134] [Q&A]
Hello!

Guys, I need a review from you. 
If you know HestiaCP NGINX templates. Check the pull request, please. 

https://github.com/hestiacp/hestiacp/pull/1973

**Comment by angrybrad** [comment]
I'm not familiar with HestiaCP, so can't comments on the project-specific syntax, but there are a few nginx location blocks that seem unnecessary.

Specifically these:

```
    location ^~ /index.php/admin {
        try_files $uri $uri/ @phpfpm_nocache;
    }

...

    location ^~ /actions {
        try_files $uri $uri/ /index.php?$query_string;
    }
```

The rest looks fine to me, though.

**Reply by anvme** [reply]
> I'm not familiar with HestiaCP, so can't comments on the project-specific syntax, but there are a few nginx location blocks that seem unnecessary.
> 
> Specifically these:
> 
> ```
>     location ^~ /index.php/admin {
>         try_files $uri $uri/ @phpfpm_nocache;
>     }
> 
> ...
> 
>     location ^~ /actions {
>         try_files $uri $uri/ /index.php?$query_string;
>     }
> ```
> 
> The rest looks fine to me, though.

oh, taken from here https://github.com/nystudio107/nginx-craft/blob/master/forge-templates/NginxFastCGICacheTemplate.conf#L126

**Reply by khalwat** [reply]
Those are needed _if_ you're using a FastCGI Cache. If you're not, they are not needed.
